lib/protoflow-blocks/src/system.rs

```diff
@@ -242,14 +251,19 @@ impl TextBlocks for System {
     }
 
     fn concat_strings_by(&mut self, delimiter: &str) -> ConcatStrings {
-        self.0.block(ConcatStrings::with_system(self, Some(delimiter.to_string())))
+        self.0.block(ConcatStrings::with_system(
+            self,
+            Some(delimiter.to_string()),
+        ))
     }
 
     fn split_string(&mut self, delimiter: &str) -> SplitString {
-        self.0.block(SplitString::with_system(self, Some(delimiter.to_string())))
+        self.0
+            .block(SplitString::with_system(self, Some(delimiter.to_string())))
     }
 
     fn split_string_whitespace(&mut self) -> SplitString {
-        self.0.block(SplitString::with_system(self, Some(r"\s+".to_string())))
+        self.0
+            .block(SplitString::with_system(self, Some(r"\s+".to_string())))
```

Comment:
@evren-okcu This is going to cause a merge conflict with both #11 and #17. Best would be to not include nonrelevant lines into PRs; that is best accomplished by use of `git add -p`.

Reply:
done